specification/security/resource-manager/Microsoft.Security/preview/2021-01-15-preview/ingestionSettings.json

```diff
@@ -0,0 +1,391 @@
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "put": {
+        "x-ms-examples": {
+          "Create an ingestion setting for subscription": {
+            "$ref": "./examples/IngestionSettings/CreateIngestionSetting_example.json"
+          }
+        },
+        "tags": [
+          "Ingestion Settings"
+        ],
+        "description": "Create setting for ingesting security data and logs to correlate with resources associated with the subscription.",
+        "operationId": "IngestionSettings_Create",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/IngestionSettingName"
+          },
+          {
+            "$ref": "#/parameters/IngestionSetting"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/IngestionSetting"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "x-ms-examples": {
+          "Delete an ingestion setting for the subscription": {
+            "$ref": "./examples/IngestionSettings/DeleteIngestionSetting_example.json"
+          }
+        },
+        "tags": [
+          "Ingestion Settings"
+        ],
+        "description": "Deletes the ingestion settings for this subscription.",
+        "operationId": "IngestionSettings_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/IngestionSettingName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The resource was deleted successfully"
+          },
+          "204": {
+            "description": "The resource was not found"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Security/ingestionSettings/{ingestionSettingName}/getToken": {
```

Comment:
This should be "listTokens" (or some other name with a list prefix) so that customers can call it from a template if they wish.  Returning one token is fine